driver/task/manager.cpp

```diff
@@ -107,7 +107,6 @@ freighter::Error task::Manager::stop() {
         task->stop();
         LOG(INFO) << "[driver] task " << task->name() << " stopped";
     }
-    tasks.clear();
```

Comment:
Why did you clear this?

Reply:
Are you asking why we cleared - i think before it had seemed to fix  an error we received on shutdown but did not address the root of the problem. It then became the cause of one of the heap corruption issues we are currently facing.